mico-core/src/test/java/io/github/ust/mico/core/TestConstants.java

```diff
@@ -63,23 +57,34 @@
     public static final String SHORT_NAME_1_MATCHER = JsonPathBuilder.buildSingleMatcher(SHORT_NAME_ATTRIBUTE, SHORT_NAME_1);
     public static final String SHORT_NAME_2_MATCHER = JsonPathBuilder.buildSingleMatcher(SHORT_NAME_ATTRIBUTE, SHORT_NAME_2);
     public static final String SHORT_NAME_3_MATCHER = JsonPathBuilder.buildSingleMatcher(SHORT_NAME_ATTRIBUTE, SHORT_NAME_3);
-    
-    public static final String DESCRIPTION = "Description Service";
-    public static final String DESCRIPTION_1 = "Description Service 1";
-    public static final String DESCRIPTION_2 = "Description Service 2";
-    public static final String DESCRIPTION_3 = "Description Service 3";
-    public static final String DESCRIPTION_ATTRIBUTE = buildAttributePath("description");
+
+    public static final String NAME = "test-name";
+    public static final String NAME_1 = "test-name";
```

Comment:
different test constants should have different values to allow testing for (in)equality!